# trio_genetic_map
## Description
The purpose of this project is to _denovo_ construct a consensus genetic map, by leveraging a high quality genome and Trio mapping

## Motivation: Why trio-based mapping?
A high-quality consensus genetic map plays a critical role in many key breeding and biotech processes, such as gene discovery, trait integration, haplotype construction, marker selection, and genotype imputation. This trio-based mapping method has four key advantages over our current method of bi-parental mapping. 
1) Foremost, significantly better map quality and map resolution. This improves our ability to map genes that are associated with traits, our efficiency in trait integration, our precision in haplotype construction and our accuracy on genotype imputation.
2) Dramatically cut the population development time and field labor cost to ZERO. This method leverages our existing resource of many thousands of trios, avoiding many years efforts of developing mapping populations, and saving labor cost, field resource and unnecessary genotyping.
3) The ability to develop more consensus genetic map without biased toward any specific germplasm. And the ability to identify key structural variations related to certain origins. 
4) The ability to know the genetic distances and varainces between any pair of Infinium markers for any part of the genome. 

## Summary
    ### There are three main elements needed in order to build a trio-based consensus map
      1. Marker information: marker squences were mapped to a target high quality reference genome to locate their positions and orders
      2. Pedigree information: a list of trios (progeny and parents)
      3. Genotype information



## Introduction
__1__. Physical map vs Genetic Map (https://www.genome.gov/genetics-glossary/Physical-Map)

   <img src="https://www.genome.gov/sites/default/files/media/images/2022-05/Physical-map.jpg">
